docs/DE0005.md

```diff
@@ -6,10 +6,10 @@ T:System.Net.Mail.SmtpClient
 
 ## Motivation
 
-`SmtpClient` does not support many modern protocols. It is compat-only. It's
+[`SmtpClient`](https://docs.microsoft.com/dotnet/api/system.net.mail.smtpclient) does not support many modern protocols. It is compat-only. It's
 great for one off emails from tools, but does not scale to modern requirements
 of the protocol.
 
 ## Recommendation
 
-Use `MailKit` or other libraries.
+Use `MailKit` or other libraries.
```

Comment:
Since you're adding other links, also add a link to MailKit (probably https://github.com/jstedfast/MailKit)?